src/schema.py

```diff
@@ -95,15 +99,104 @@ class Meta:
         model = CapacityModel
 
 
+# MARK: - User
+
+
+class User(SQLAlchemyObjectType):
+    class Meta:
+        model = UserModel
+
+
+class UserInput(graphene.InputObjectType):
+    net_id = graphene.String(required=True)
+    giveaway_id = graphene.Int(required=True)
+
+
+# MARK: - Giveaway
+
+
+class Giveaway(SQLAlchemyObjectType):
+    class Meta:
+        model = GiveawayModel
+
+    user_ids = graphene.List(lambda: User)
+
+    def resolve_user_ids(self, info):
+        query = User.get_query(info=info).filter(UserModel.giveaway_id == self.id)
+        return query
+
+
+# MARK: - Giveaway
+
+
+class GiveawayInstance(SQLAlchemyObjectType):
+    class Meta:
+        model = GiveawayInstanceModel
+
+
 # MARK: - Query
 
 
 class Query(graphene.ObjectType):
     gyms = graphene.List(Gym)
+    users_by_giveawayid = graphene.List(User, id=graphene.Int())
```

Comment:
```suggestion
    get_all_gyms = graphene.List(Gym, description="Get all gyms.")
    get_users_by_giveaway_id = graphene.List(User, id=graphene.Int(), description="Get all users given a giveaway ID.")
```

Frontend will need to refactor a bit but because these are also field resolvers, it's a lot more clear to prepend them with "get". When I was working on Volume, this was the convention and it makes it a lot more clear. We should also add a description since Graph iQL can use this for documentation purposes. See the screenshot below. Also, add an underscore in `giveawayid` so that GraphQL can convert this to camelCase.

<img width="341" alt="image" src="https://github.com/cuappdev/uplift-backend/assets/75594943/c08c69f1-912e-4fee-a855-45bcc011b6d4">